public/index.html

```diff
@@ -19,7 +19,14 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>MusicBox</title>
```

Comment:
Anyone know if there's a better way to do this with NPM, or JS imports? It feels hacky, but it does work/load the font.